examples/02_decoding/plot_haxby_glm_decoding.py

```diff
@@ -104,84 +103,62 @@
 # In a jupyter notebook, the report will be automatically inserted, as above.
 # We have several other ways to access the report:
 
-# report  # This report can be viewed in a notebook
 # report.save_as_html('report.html')
 # report.open_in_browser()
 
-#############################################################################
-# Transform the maps to an array of values
-# ----------------------------------------
-from nilearn.input_data import NiftiMasker
-
-# no need to standardize or smooth the data
-masker = NiftiMasker(mask_img=haxby_dataset.mask, memory='nilearn_cache',
-                     memory_level=1)
-X = masker.fit_transform(z_maps)
-
-#############################################################################
-# Build the decoder
-# ------------------
-# Define the prediction function to be used.
-# Here we use a Support Vector Classification, with a linear kernel
-from sklearn.svm import SVC
-
-svc = SVC(kernel='linear')
-
-# Define the dimension reduction to be used.
-# Here we use a classical univariate feature selection based on F-test,
-# namely Anova. When doing full-brain analysis, it is better to use
-# SelectPercentile, keeping 5% of voxels
-# (because it is independent of the resolution of the data).
-from sklearn.feature_selection import SelectPercentile, f_classif
-
-feature_selection = SelectPercentile(f_classif, percentile=5)
-
-# We have our classifier (SVC), our feature selection (SelectPercentile),and now,
-# we can plug them together in a *pipeline** that performs the two operations
-# successively:
-from sklearn.pipeline import Pipeline
-
-anova_svc = Pipeline([('anova', feature_selection), ('svc', svc)])
-
-#############################################################################
-# Obtain prediction scores via cross validation
-# -----------------------------------------------
-from sklearn.model_selection import cross_val_score, LeaveOneGroupOut
-
-# Define the cross-validation scheme used for validation.
-# Here we use a LeaveOneGroupOut cross-validation on the session group
-# which corresponds to a leave-one-session-out
-cv = LeaveOneGroupOut()
-
-# Compute the prediction accuracy for the different folds (i.e. session)
-cv_scores = cross_val_score(anova_svc, X, condition_idx, cv=cv, groups=session_idx)
-
-# Return the corresponding mean prediction accuracy
-classification_accuracy = cv_scores.mean()
-chance_level = 1. / len(np.unique(condition_idx))
-
-# Print the results
+##############################################################################
+# Build the decoding pipeline
+# ----------------------------
+# To define the decoding pipeline we use Decoder object, we choose :
+#
+# * a prediction model, here a Support Vector Classifier, with a linear kernel
+#
+# * the mask to use, here a ventral temporal ROI in the visual cortex
+#
+# * although it usually helps to decode better, z-maps time series don't need
+#    to be rescaled to a 0 mean, variance of 1 so we use standardize=False.
```

Comment:
I wonder if the number of spaces here is right? It seems to me that there are 4 spaces, while there should be 3, to match the indent of the bullet-point list